src/functionalTest/java/uk/gov/hmcts/reform/iahearingsapi/RecordAdjournmentDetailsPreparerFunctionalTest.java

```diff
@@ -21,6 +22,7 @@
 
 @Slf4j
 @ActiveProfiles("functional")
+@Disabled
```

Comment:
Let's create at least a tech debt task for all these disabled tests

Reply:
I have now enabled the tests

Reply:
Thanks a lot